container/dangerzone.py

```diff
@@ -261,7 +261,14 @@ def pdftoppm_progress_callback(line: bytes) -> None:
                 num_pages = int(num_pages_str)
                 page = int(page_str)
             except ValueError as e:
-                raise RuntimeError("Conversion from PDF to PPM failed") from e
+                if line.decode().startswith(
+                    "Syntax Warning"
+                ) or line.decode().startswith("Syntax Error"):
+                    # Ignore syntax warnings and errors.
+                    # pdftoppm produces a valid document regardless of these errors
+                    return
```

Comment:
I think you've mentioned it somewhere, but I don't see the comment now. Since `pdftoppm` may print arbitrary error messages in stderr, we should just discard such lines (they will be included in the logs after all), and simply rely on its return code.

Reply:
I agree.

Reply:
Resolved via [cf0c724](https://github.com/freedomofpress/dangerzone/pull/338/commits/cf0c724e6bab5e3cf48f22b52c12d0908df89aa8).